src/caf/distribute/furness.py

```diff
@@ -157,3 +161,234 @@ def doubly_constrained_furness(
         )
 
     return furnessed_mat, iter_num + 1, cur_rmse
+
+
+# pylint: disable=too-many-arguments, too-many-locals
+def furness_pandas_wrapper(
+    seed_values: pd.DataFrame,
+    row_targets: pd.DataFrame,
+    col_targets: pd.DataFrame,
+    max_iters: int = 2000,
+    seed_infill: float = 1e-3,
+    normalise_seeds: bool = True,
+    tol: float = 1e-9,
+    idx_col: str = "model_zone_id",
+    unique_col: str = "trips",
+    round_dp: int = 8,
+    unique_zones: list[int] | None = None,
+    unique_zones_join_fn: Callable = operator.and_,
+) -> tuple[pd.DataFrame, int, float]:
```

Comment:
Pylint isn't happy about too-many-positional-arguments and I would agree, I think just switching all the optional ones to keyword only would be good.
```suggestion
def furness_pandas_wrapper(
    seed_values: pd.DataFrame,
    row_targets: pd.DataFrame,
    col_targets: pd.DataFrame,
    *,
    max_iters: int = 2000,
    seed_infill: float = 1e-3,
    normalise_seeds: bool = True,
    tol: float = 1e-9,
    idx_col: str = "model_zone_id",
    unique_col: str = "trips",
    round_dp: int = 8,
    unique_zones: list[int] | None = None,
    unique_zones_join_fn: Callable = operator.and_,
) -> tuple[pd.DataFrame, int, float]:
```

Above is a breaking change so might break wherever this function is called depending if they used positional arguments for everything.